pkgs/build-support/bintools-wrapper/default.nix

```diff
@@ -53,7 +53,8 @@ let
   dynamicLinker =
     /**/ if libc == null then null
     else if targetPlatform.libc == "musl"             then "${libc_lib}/lib/ld-musl-*"
-    else if targetPlatform.libc == "bionic"           then "/system/bin/linker"
+    else if (targetPlatform.libc == "bionic" && targetPlatform.parsed.cpu.bits == 32) then "/system/bin/linker"
+    else if (targetPlatform.libc == "bionic" && targetPlatform.parsed.cpu.bits == 64) then "/system/bin/linker64"
```

Comment:
```suggestion
    else if (targetPlatform.libc == "bionic" && targetPlatform.is32bit) then "/system/bin/linker"
    else if (targetPlatform.libc == "bionic" && targetPlatform.is64bit) then "/system/bin/linker64"
```

Reply:
Why got this reverted?

Reply:
what do you mean? It wasn't reverted... I don't think.

Reply:
Oh, I thought you reverted it when force pushing. I think we should apply that suggestion if it works.